contracts/contracts/RaidenMicroTransferChannels.sol

```diff
@@ -281,6 +288,44 @@ contract RaidenMicroTransferChannels {
         require(token.transferFrom(msg.sender, address(this), _added_deposit));
     }
 
+    /// @notice Allows channel receiver to withdraw tokens.
+    /// @param _open_block_number The block number at which a channel between the
+    /// sender and receiver was created.
+    /// @param _balance The amount of tokens owed by the sender to the receiver.
+    /// Has to be smaller or equal to the channel deposit.
+    /// @param _balance_msg_sig The balance message signed by the sender.
+    function withdraw(
```

Comment:
Why is the receiver forced to withdraw the entire owed balance to him here? Why not provide the option to just withdraw part of it?

Reply:
The receiver just has to provide a balance proof signed by the sender. 
In theory, it can be any balance proof - even an older one. In practice, the current microraiden server only stores the last balance proof. 

So, the comment might be misleading. I can change to
```
/// @param _balance Partial or total amount of tokens owed by the sender to the receiver.
/// Has to be smaller or equal to the channel deposit. Has to match the balance value
/// from `_balance_msg_sig` - the balance message signed by the sender.   
```

Reply:
yep that sounds better. The microraiden server implementation may change at some point and allow for partial withdrawal without any contract changes